authorities.md

```diff
@@ -125,22 +121,37 @@ sample for an authority /server/api/integration/authorities/common_types/entries
         "id": "Dataset",
         "display": "Dataset",
         "value": "Dataset",
-        "otherInformation": {},
-        "type": "authority"
+        "metadata": {},
+        "type": "authority",
+        "_links": {
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/authorities/common_types/entryValues/Dataset"
+          }
+        }
       },
       {
         "id": "Image, 3-D",
         "display": "Image, 3-D",
         "value": "Image, 3-D",
-        "otherInformation": {},
-        "type": "authority"
+        "metadata": {},
```

Comment:
metadata has a special meaning in dspace so this could be confusing. The idea here is that the authority can expose raw information not necessary "metadata" and to deal with them additional knowledge is usually required to the client.

An example of additional information that could result useful and is not a metadata are metrics value, number of items in the repository or in general number of publications for a researcher etc

Reply:
The information being included here exactly matches the definition of metadata. It can also be represented similar to item metadata since the information currently included here is e.g.:
* An ORCID identifier
* A description
* A parent reference
By ensuring this is also metadata, the UI can render the search results in a manner consistent with other lists

The information you're referencing here is not included in the authority functionality and don't seem to be relevant to include in an authority record.

Reply:
@benbosman : As above, I'm confused why this PR is still modifying the `authorities` REST Contract, as I thought the decision was to change this PR to only create the Read-Only `external-sources` endpoint.  Maybe we should move this discussion to a separate PR to update the `authorities` endpoint based on current behavior?